### ISSUE TITLE
Incompatible with latest Stripe (10.3)

### ISSUE THREAD

**Issue by Papoel** [FabienPapet/stripe-bundle#10]
I am trying to add this bundle to my project but it seems that the latest version of Stripe is not usable.
Below is the error from the terminal

```bash
Information from https://repo.packagist.org : #StandWithUkraine
Using version ^0.8.1 for fpt/stripe-bundle
./composer.json has been updated
Running the update of composer fpt/stripe-bundle
Loading composer repositories with package information
Updated dependencies
Your requirements could not be resolved into a set of installable packages.

 Problem 1
    - The root file composer.json requires fpt/stripe-bundle ^0.8.1 -> satisfiable by fpt/stripe-bundle [0.8.1].
    - fpt/stripe-bundle 0.8.1 requires stripe/stripe-php ^v7.121.0|^v8.0.0|^v9.0.0 -> found stripe/stripe-php [v7.121.0, ..., v7.128.0, v8.0.0, ..., v8.12.0, v9.0.0, ..., v9.9.0] but it conflicts with your root composer.json require (^10.3).

Use the --with-all-dependencies (-W) option to allow upgrades, downgrades and removals of packages currently locked to specific versions.
You can also try to rerun compose require with an explicit version constraint, e.g. "compose require fpt/stripe-bundle:*" to see if a version is installable, or "compose require fpt/stripe-bundle:^2.1" if you know which one you need.
The installation failed and the ./composer.json and ./composer.lock files reverted to their original content.
```

```json
Composer.json:
"require": {
    ....
    "stripe/stripe-php":"^10.3",
}
```

Thanks.

**Comment by xorgxx** [comment]
+1

**Comment by FabienPapet** [comment]
Hello, @Papoel  thank you for reporting this issue. I just merged a fix to allow v10 and publish 0.9, can you please have a look ?

**Comment by Papoel** [comment]
Hi @FabienPapet sorry for answering so late, I haven't had time to look yet but I will do that soon. Thanks for the consideration.
